src/components/cart/components/cartCheckout/CartCheckout.js

```diff
@@ -0,0 +1,58 @@
+// Libraries
+import PropTypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+// constants
+import { LOCAL_STORAGE_KEY } from "./cartCheckout.constants";
+
+// helpers
+import { saveItem as saveItemToLocalStorage } from "./helpers/localStorage";
+import getTotalAmount from "./helpers/getTotalAmount";
+import getCartItems from "./helpers/getItems";
+
+// utility
+import getFormattedAmount from "../../../../utility/getFormattedAmount";
+
+// css
+import "./cartCheckout.css";
+
+function CartCheckout(props) {
```

Comment:
Can't we calculate the amount in parent component and then pass the amount as a prop and for checkout button directly call the handler where the state has been kept and do whatever we wanna do their itself?